src/pages/SocialGraph.vue

```diff
@@ -270,9 +290,31 @@ export default {
     updateTitle() {
       this.title = `The ${this.type}s of ${this.inputString}`;
     },
-    exportPageAddressList() {
-      const accounts = this.currentPageData.map((c) => c.account);
-      downloadAsFile(accounts.join('\n'), `${this.type}_of_${this.inputString}_page_${this.currentPage}.csv`, 'text/csv');
+    async fetchAllPageData() {
+      this.allPageData = [];
+      this.isLoadingAllPageData = true;
+      let i = INITIAL_PAGE;
+      let isPageEnd = false;
+      while (!isPageEnd) {
+        // eslint-disable-next-line no-await-in-loop
+        const data = await this.fetchPageData(i);
+        isPageEnd = !(data && data.length);
+        if (!isPageEnd) this.allPageData = this.allPageData.concat(data);
+        i += 1;
+      }
+      this.allPageData.sort(
+        (a, b) => b.totalValue - a.totalValue,
+      );
+
+      // TODO: Fix indexer API to not return duplicated addresses
+      this.allPageData = this.allPageData
+        .filter((c, index) => this.allPageData.indexOf(c) === index);
```

Comment:
```suggestion
      this.allPageData = [...new Map(this.allPageData.map(c => [c.account, c]).values()];
```

Reply:
sorcery... updated

Reply:
If two or more items in the `allPageData` has the same `account`, will the item with the lowest `totalValue` remain in the `allPageData`, right?

Reply:
the strange thing is indexer returns the exact same content more than once for some address in different pages, so the dropped version should be same as kept one